torch/quantization/fx/quantize.py

```diff
@@ -511,7 +511,7 @@ def load_arg(a):
 
         to_be_removed = []
         for name, module in model.named_modules():
-            if is_activation_post_process(module):
+            if 'activation_post_process_' in name.split('.')[-1]:
```

Comment:
why is this changed?

Reply:
This is for the observer modules that are an attr of the fakequant module. If we just check the module type then it returns true and then fails when we try to call `delattr` below.

Reply:
I see, maybe make more sense to fix this by avoiding checking the submodule of fakequantize module?

Reply:
The submodule (i.e observer) shows up in `model.named_modules()` as well. So we need some additional check to not add that in the removed list.

Reply:
yes, I think so, feel free to move this to a function